board/netconf/rootfs/libexec/infix/cli-pretty

```diff
@@ -207,10 +212,43 @@ class Iface:
         row += f"{self.phys_address:<{Pad.data}}"
         print(row)
 
+    def pr_proto_br(self, br_vlans):
+        data_str = ""
+
+        row = f"{'bridge':<{Pad.proto}}"
+
+        if self.oper_status == "down":
+            row += Decore.red(f"{'DOWN':<{Pad.state}}")
```

Comment:
```suggestion
        if self.oper_status != "up":
            row += Decore.red(f"{self.oper_status.upper()':<{Pad.state}}")
```
I think that if the interface is not fully operational, for any reason, then I think that should have precedent over the bridge state. I.e., if the interface reports `lower-layer-down`, then that interface will never show any other bridge state than `disabled`.